packages/core/src/App/Containers/RealAccountSignup/change-account-currency.jsx

```diff
@@ -19,26 +19,13 @@ const ChangeAccountCurrency = ({
     client_currency,
     current_currency_type,
     current_fiat_currency,
-    closeModal,
```

Comment:
I suppose we can not pass 'closeModal' into <ChangeAccountCurrency/>  in the place where it is rendered.
And also we can remove 'closeModal' from propTypes in the end of this file.